dm/dm/config/subtask.go

```diff
@@ -415,7 +416,7 @@ func (c *SubTaskConfig) Adjust(verifyDecryptPassword bool) error {
 		} else {
 			dirSuffix = c.Name
 		}
-		newDir, err := storage.AdjustPath(c.LoaderConfig.Dir, "."+dirSuffix)
+		newDir, err := storage.AdjustPath(c.LoaderConfig.Dir, string(filepath.Separator)+dirSuffix)
```

Comment:
`storage.AdjustPath` has no change and may be we can use in future, not only add `/xxx`.